internal/commands/util.go

```diff
@@ -19,7 +19,6 @@ func ping(ctx context.Context, app *app.App, channelID string) {
 		app.Logger.Error(
 			ctx,
 			"command/util.ping postMessage error",
-			log.NewValue("channelID", channelID),
```

Comment:
Why removing the channelID field from this log but not adding a logWriter with it?